### PR TITLE
Toolbox: add NFD undeploy script

### PR DESCRIPTION
**Pull request by kpouget**
This PR goes on top of https://github.com/openshift-psap/ci-artifacts/pull/51

* 146fa39 - toolbox: nfd/undeploy_from_operatorhub.sh: new script / ansible role / functionality

* a691e81 - toolbox: move deploy_nfd_from_operatorhub.sh to nfd/deploy_from_operatorhub.sh
^^^ to have a more logical organization of the scripts

### CODE REVIEW

**Comment by kpouget** [comment]
/hold
comes after https://github.com/openshift-psap/ci-artifacts/pull/51

**Comment by kpouget** [comment]
/unhold
reorganized commits to make PR fully independent

File: roles/nv_gpu/tasks/uninstall_nfd.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  register: operator_csv_name
+  ignore_errors: true
+
+- name: Delete the ClusterServiceVersion of the NFD Operator
+  with_items: "{{ operator_csv_name.stdout_lines }}"
+  shell: "oc delete {{ item }} -n openshift-operators"
+  ignore_errors: true
+  when: operator_csv_name.rc == 0
+
+- name: Delete the clusterPolicy CR of the NFD Operator
```
Comment (ArangoGutierrez) [inline roles/nv_gpu/tasks/uninstall_nfd.yml:18]:
```suggestion
- name: Delete the NodeFeatureDiscovery CR of the NFD Operator
```

Comment (ArangoGutierrez) [reply roles/nv_gpu/tasks/uninstall_nfd.yml:18]: `clusterPolicy` is the name of the Nvidia CR

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: roles/nv_gpu/tasks/uninstall_nfd.yml (hunk 1 above)
Comment (kpouget) [reply roles/nv_gpu/tasks/uninstall_nfd.yml]: the copy-and-paste was too fast :#

fixed

File: roles/nv_gpu/tasks/uninstall_nfd.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+- name: Delete the OperatorHub subscription for the NFD Operator
+  command: oc delete -f "{{ nfd_operator_operatorhub_sub }}"
+  ignore_errors: true
+
```
Comment (ArangoGutierrez) [inline roles/nv_gpu/tasks/uninstall_nfd.yml:5]: nit 2 white lines

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]
last nit

File: roles/nv_gpu/tasks/uninstall_nfd.yml (hunk 2 above)
Comment (kpouget) [reply roles/nv_gpu/tasks/uninstall_nfd.yml:5]: fixed

**Pull request merged**